app/Event/EventControllerProvider.php

```diff
@@ -54,7 +54,6 @@ public function connect(Application $app): ControllerCollection
 
         $controllers->put('/{cdbid}/booking-info/', 'event_editing_controller:updateBookingInfo');
         $controllers->put('/{cdbid}/contact-point/', 'event_editing_controller:updateContactPoint');
-        $controllers->put('/{cdbid}/organizer/{organizerId}/', 'event_editing_controller:updateOrganizer');
```

Comment:
This one should probably also be removed? https://github.com/cultuurnet/udb3-silex/blob/master/app/Place/PlaceControllerProvider.php#L37

Reply:
+ The place equivalent of the editing service